test/verify/check-kdump

```diff
@@ -82,9 +82,9 @@ class TestKdump(KdumpHelpers):
             # crashkernel command line not set
             b.wait_visible(".pf-c-switch__input:disabled")
             # right now we have no memory reserved
-            b.mouse("span + .popover-ct-kdump", "mouseenter")
+            b.mouse("span + div > .popover-ct-kdump", "mouseenter")
```

Comment:
Please squash "tests: adjust tests to expect the PF5 changes on dom structure" into "move to pf5" commit